public class Foo {
  void foo() {
    try {
    } finally {
      try {
        int x = 2;
      } catch (Exception e) {
      }
    }
  }
}

